.pre-commit-config.yaml

```diff
@@ -1,19 +1,32 @@
-# Rerun `pre-commit install` after every update of this file
 repos:
--   repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.77.0
+-   repo: local
+    hooks:
+    -   id: terraform_format
+        name: terraform_format
+        entry: --entrypoint /src/hooks/terraform_format.sh hashicorp/terraform:1.5.2

```

Comment:
Can we use inline command call or the bash file with one command call is needed?

Reply:
Sadly, unline command did not work for me.

Reply:
I believe that something goes wrong when pre-commit passes docker arguments. I was not able to find it out in time.